packages/babel-parser/test/fixtures/typescript/class/declare-accessor/input.ts

```diff
@@ -0,0 +1,4 @@
+class Foo {
+  declare get foo()
+  declare set foo(v)
+}
```

Comment:
Can you add a test case when `get/set` is a property name?

```ts
class Foo {
  declare get: string;
  declare set: string;
}
```

They shout not throw because they are not accessors.

Reply:
Nice, that doesn't work because the check is done too early, I'll fix in the weekend.